### PR TITLE
Clean some code.

### PR DESCRIPTION
**Pull request by cedric-anne**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

- Remove useless docblocks when defined in parent.
- Remove unused vars.
- Fix some docblock and types.

### CODE REVIEW

File: inc/config.class.php [hunk 1]
```diff
@@ -665,20 +650,19 @@ function showFormAuthentication() {
...
     *
-    * @return Nothing (display)
+    * @return void
    **/
    function showFormDBSlave() {
       global $DB, $CFG_GLPI, $DBslave;
 
-      if (!Config::canUpdate()) {
-         return false;
+      if (!DBConnection::isDBSlaveActive() || !Config::canUpdate()) {
```
Comment (trasher) [inline inc/config.class.php]: I do not agree with that change

File: inc/db.function.php [hunk 2]
```diff
@@ -545,10 +545,10 @@ function exportArrayToDB($TAB) {
 /**
  * Import an array encoded in a simple field in the database
  *
- * @param $DATA data readed in DB to import
+ * @param string $data data readed in DB to import
```
Comment (trasher) [inline inc/db.function.php]: Param is uppercase in the code

File: inc/config.class.php (hunk 1 above)
Comment (cedric-anne) [reply inc/config.class.php]: The form tab is displayed only if `!DBConnection::isDBSlaveActive()` (see https://github.com/glpi-project/glpi/blob/42477190a201a1f5e49703be1c9af3d6fe0045cb/inc/config.class.php#L2082). Adding this test was just made to prevent failure on form display as `$DBslave` will be `null` if `!DBConnection::isDBSlaveActive()`. As this form should not be displayed in this case, I can remove the test if you want.

I found where was removed the test on `$active` var -> https://github.com/glpi-project/glpi/commit/439c40a2a253a9838320c1a8677622f02b16a7a9

File: inc/db.function.php (hunk 2 above)
Comment (cedric-anne) [reply inc/db.function.php]: Fixed. Mistake was made as I copied/paste well formed documentation from `DbUtils` class.
